### PR TITLE
feat: messageEvent should include timestamp

### PR DESCRIPTION
**Pull request by tcolgate**

### CODE REVIEW

**Comment by kjin** [comment]
@tcolgate Also, can you specify in the JSDoc that timestamp should be in milliseconds?

File: packages/opencensus-core/src/trace/model/span-base.ts [hunk 1]
```diff
@@ -169,11 +169,13 @@ export abstract class SpanBase implements types.Span {
   /**
    * Adds a message event to the span.
    * @param type The type of message event.
+   * @param timestamp The time the message occurred.
    * @param id An identifier for the message event.
    */
-  addMessageEvent(type: string, id: string) {
+  addMessageEvent(type: string, timestamp: number, id: string) {
```
[the remark this answers is not in the document]
Comment (tcolgate) [reply packages/opencensus-core/src/trace/model/span-base.ts]: This sounds like a good idea, should be apply the same thinking to addAnnotation though?

[the remark this answers is not in the document]
Comment (kjin) [reply packages/opencensus-core/src/trace/model/span-base.ts]: Yes, if it is already there would you mind switching that?

**Comment by kjin** [comment]
@tcolgate Thanks!

**Pull request merged**